scripts/pvscheck.sh

```diff
@@ -1,12 +1,14 @@
-#!/bin/sh
+#!/usr/bin/env bash
```

Comment:
I see.
Should we then use the `foo() {(bar=1)}` pattern instead? (used elsewhere already)